src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -730,8 +593,11 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // - hr: an  HRESULT describing the warning
     // Return Value:
     // - <none>
-    winrt::fire_and_forget TermControl::_RendererWarning(const HRESULT hr)
+    winrt::fire_and_forget TermControl::_RendererWarning(const IInspectable& /*sender*/,
+                                                         const Control::RendererWarningArgs& args)
     {
+        const HRESULT hr = static_cast<HRESULT>(args.Result());

```

Comment:
```suggestion
        const auto hr = static_cast<HRESULT>(args.Result());
```

nit: repeated type left/right